nttt/utilities.py

```diff
@@ -24,6 +24,13 @@ def find_snippet(s, start_string, end_string):
     else:
         return None
 
+def find_replace(src, dst, find, replace):
```

Comment:
The method should no longer be used anywhere. Please double check it but it should be safe to remove it